src/auth/credential-provider.ts

```diff
@@ -24,35 +24,41 @@ export interface CredentialProvider {
 }
 
 /**
- * Reads and parses a momento auth token stored as an environment variable.
+ * Encapsulates arguments for instantiating a StringMomentoTokenProvider
+ */
+export interface StringMomentoTokenProviderProps {
+  /**
+   * authToken the momento auth token
+   */
+  authToken: string;
+  /**
+   * optionally overrides the default controlEndpoint
+   */
+  controlEndpoint?: string;
+  /**
+   * optionally overrides the default cacheEndpoint
+   */
+  cacheEndpoint?: string;
+}
+
+/**
+ * Reads and parses a momento auth token stored in a String
  * @export
  * @class EnvMomentoTokenProvider
  */
-export class EnvMomentoTokenProvider implements CredentialProvider {
+export class StringMomentoTokenProvider implements CredentialProvider {
```

Comment:
You'll need to change the comment on line 47 to reflect the correct class name.

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/204/commits/c0c2ce92ab102d408b1497ebb4091cb22a8576c5